### PR TITLE
event: publish early June devenv sprint

### PR DESCRIPTION
**Pull request by j8xixo12**
Publish 06/05 devenv sprint website.

### CODE REVIEW

File: content/pages/sprint/2021/06early-devenv.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+possible to do it all by a single team.  External dependencies are necessary.
+However, because of the requirements of accuracy and performance, the
+dependencies are usually at the source level.  Building dependencies from
+source is the norm.  The project devenv is made to help the source-level
+dependency management and alignment for the runtime among developers.
+
+Registration
+------------
+
+Please RSVP at:
```
Comment (chestercheng) [inline content/pages/sprint/2021/06early-devenv.rst:35]: This time we don't have rsvp url. How about put sciwork gather.town url on here?

Comment (yungyuc) [reply content/pages/sprint/2021/06early-devenv.rst:35]: I second the idea of using the gather.town link.

Comment (j8xixo12) [reply content/pages/sprint/2021/06early-devenv.rst]: Agree !!

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
LGTM.  Merging.

**Comment by j8xixo12** [comment]
I modified registration url to gather.town.

**Pull request merged**